libcontainer/cgroups/fscommon/utils.go

```diff
@@ -83,9 +83,9 @@ func GetValueByKey(path, file, key string) (uint64, error) {
 
 	lines := strings.Split(content, "\n")
 	for _, line := range lines {
-		arr := strings.Split(line, " ")
-		if len(arr) == 2 && arr[0] == key {
-			val, err := ParseUint(arr[1], 10, 64)
+		k, v, ok := strings.Cut(line, " ")
+		if ok && k == key {
```

Comment:
Not sure if  relevant (and _probably_ new code is more correct?); previously, we would ignore cases with more than 2 fields. I.e., `key foo bar` would be ignored; in the new code, we'd try to parse `foo bar` as value.

That said; see my other comment; this would potentially be an issue if we try to parse something that may currently have 2 cols, and in future 3 cols; not sure if that should _error_ or silently _ignore_